t/http-router.t

```diff
@@ -1460,14 +1460,14 @@ throws-like { bad-request }, X::Cro::HTTP::Router::OnlyInHandler, what => 'bad-r
     my $req = Cro::HTTP::Request.new(method => 'GET', target => '/');
     $source.emit($req);
     given $responses.receive -> $r {
-        is body-text($r), "<HTML></HTML>\n", 'Static index is fine';
+        like body-text($r), rx{ '<HTML></HTML>' \r? \n }, 'Static index is fine';
```

Comment:
In a regex, `\n` is logical, so will match `\r\n` anyway. Since `\r\n` is a single grapheme, the `\r` on its own here will always be failing to match, backtracking, and then having the logical `\n` eat it. So the `\r?` part of this isn't needed. Otherwise, looks completely fine.

Reply:
Ah! Didn't know that. will fix it :-)